src/components/StatusItem.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div
+    class="keyboard-status"
+    :class="keyboardItem.passed ? 'passed' : 'failed'"
+    :title="lastTestedTitle"
+    @click="$emit('show-error-pane', keyboardItem.key)"
+  >
+    {{ vendorName }}
+    <br />
+    {{ keyboardName }}
+  </div>
+</template>
+
+<script>
+import { format } from 'timeago.js';
+
+export default {
+  name: 'StatusItem',
+  components: {},
+  props: ['keyboardItem'],
+  computed: {
+    keyboardName() {
+      let firstSlash = this.keyboardItem.key.indexOf('/');
```

Comment:
To make this a bit less ugly, try using
```javascript
const { key } = this.keyboardItem;
```
Then you can just use key everywhere instead of the longer path.